packages/bbs-signatures/tests/bbs-signatures.e2e.test.ts

```diff
@@ -1,4 +1,4 @@
-import type { W3cCredentialRepository } from '../../core/src/modules/vc/repository'
+import type { W3cCredentialsRepository } from '../../core/src/modules/vc/repository'
```

Comment:
We keep the services and repositories singular (It's also CredentialRepository). It's a bit inconsistent, but we've had discussions at the time and that's what we landed on. This would be a bigger thing to change instead of `VC` vc `Credentials`